api/src/modules/users/users.controller.ts

```diff
@@ -68,6 +68,18 @@ export class UsersController {
     return this.service.serialize(results.data, results.metadata);
   }
 
+  @ApiOperation({ description: 'Create new user' })
+  @ApiCreatedResponse({ description: 'User created successfully' })
+  @ApiOkResponse({
+    type: User,
+  })
+  @Post()
```

Comment:
Should this route be more defined, like: /users/create ?

Reply:
what would be the upside of that vs current implementation?

Reply:
I feel like a endpoint defined us 
`user/create`
is more meaningful than
'user'

Reply:
In most REST API practices , POST , GET , DELETE ...  define what endpoint does to a resource .POST is creating new resource  so I think adding extra /create is redundant here  , same way as DELETE /{resource}/id  is used instead of DELETE /{resource}/id/delete . @alexeh please let me know what you think about this

Reply:
My comment was mostly because of the spec of adding a create user endpoint, but after your reasoning I changed my mind.
Thank you for the detailed explanation

Reply:
+1 @asuramaru1